src/components/Loader/__stories__/Loader.stories.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import {Meta, Story} from '@storybook/react/types-6-0';
+import {Loader, LoaderProps} from '../Loader';
+
+export default {
+    title: 'Components/Loader',
+    component: Loader,
+} as Meta;
+
+const DefaultTemplate: Story<LoaderProps> = (args) => <Loader {...args} />;
+export const Default = DefaultTemplate.bind({});
+
+const SizeTemplate: Story<LoaderProps> = (args: any) => {
```

Comment:
Here args should be infered as LoaderProps

Reply:
Yeap, any is unnecessary